blobfuse/blobfuse.cpp

```diff
@@ -348,22 +376,22 @@ int read_config(const std::string configFile)
         config_options.authType = get_auth_type();
     }
     
-    if(config_options.accountName.empty())
-    {
-        syslog (LOG_CRIT, "Unable to start blobfuse. Account name is missing in the config file.");
-        fprintf(stderr, "Unable to start blobfuse. Account name is missing in the config file.\n");
-        return -1;
-    }
-    else if(config_options.containerName.empty())
-    {
-        syslog (LOG_CRIT, "Unable to start blobfuse. Container name is missing in the config file.");
-        fprintf(stderr, "Unable to start blobfuse. Container name is missing in the config file.\n");
-        return -1;
-    }
-    else
-    {
-        return 0;
+    if (!config_options.enableGen1) {
+        if(config_options.accountName.empty())
```

Comment:
accountName cannot be empty for gen1 also. Container name can be empty but if it is not empty it should get appended at the end for the resoruceid. adlsgen1 folder is treated like blob container